src/system.ts

```diff
@@ -37,6 +50,11 @@ export const bootstrapCache = async () => {
   if (PREFETCH_CONTRACTS) {
     await prefetchContracts();
   }
+
+  if (SAVE_CACHE_TO_S3) {
+    // save the cache on a 3 hour interval
+    setInterval(saveCacheToS3, 3 * 60 * 60 * 1000);
```

Comment:
nit: cacheIntervalMs const or similar

Reply:
may be good to set to an env var for infra configurability

Reply:
[8f05104](https://github.com/ar-io/arns-service/pull/108/commits/8f05104f54bcd1462b28598ed1e746adef829758)